### PR TITLE
add iotex explorer info

### PR DESCRIPTION
**Pull request by ququzone**

### CODE REVIEW

File: _data/chains/eip155-4689.json [hunk 1]
```diff
@@ -15,5 +15,9 @@
   "infoURL": "https://iotex.io",
   "shortName": "iotex-mainnet",
   "chainId": 4689,
-  "networkId": 4689
+  "networkId": 4689,
+  "explorers": [{
+    "name": "iotexscan",
```
Comment (ligi) [inline _data/chains/eip155-4689.json:20]:
```suggestion
    "name": "iotexscan",
    "standard": "EIP3091",
```

File: _data/chains/eip155-4690.json [hunk 2]
```diff
@@ -16,5 +16,9 @@
   "infoURL": "https://iotex.io",
   "shortName": "iotex-testnet",
   "chainId": 4690,
-  "networkId": 4690
+  "networkId": 4690,
+  "explorers": [{
+    "name": "testnet iotexscan",
```
Comment (ligi) [inline _data/chains/eip155-4690.json:21]:
```suggestion
    "name": "testnet iotexscan",
    "standard": "EIP3091",
```

**Review by ligi — CHANGES_REQUESTED** [review]
the "standard" field must be present

File: _data/chains/eip155-4689.json (hunk 1 above)
Comment (ququzone) [reply _data/chains/eip155-4689.json:20]: Thanks, iotexscan now support EIP3091

**Review by ligi — APPROVED** [review]

**Pull request merged**